examples/bank_account/bank_account.py

```diff
@@ -16,7 +16,7 @@ def info(self):
     def deposit(self, amount):
         """ deposit money """
         if amount > 0:
-            self._balance += amount - self._calc_commission_rate(self._hasCommissionDiscount)
+            self._balance -= amount - self._calc_commission_rate(self._hasCommissionDiscount)
```

Comment:
Revert the changes in the deposit and withdraw methods. The deposit method should increase the balance and the withdraw method should decrease the balance. [important]